<launch>
    <arg name="robot_name" default="deli_robot" />
    <group ns="$(arg robot_name)">
        <node pkg="autopilot" type="autopilot_node.py" name="autopilot_node" output="screen">
            <remap from="map" to="odometry/map"/>
            <remap from="destination" to="destList_array"/>
            <remap from="obstacles" to="line_segments"/>
            <remap from="objects" to="zed_node/obj_det/objects"/>
            <remap from="path" to="goalPosition"/>
        </node>
        <param name="test_status" value="True"/>
        <param name="map_status" value="False"/>
        <param name="test_x" value="20"/>
        <param name="test_y" value="0"/>

        <param name="resolution" value="20"/>
        <param name="zoom" value="6"/>
        <param name="size" value="1"/>

        <param name="obstacle_force" value="2"/>
        <param name="obstacle_range" value="3"/>
        <param name="attractive_force" value="5"/>
        <param name="repulsive_force" value="10"/>
        <param name="slope_force" value="5"/>

        <param name="step" value="1"/>
        <param name="check" value="5"/>

        <param name="tangen_depth" value="10"/>
        <param name="tau" value="0.8"/>
        <param name="degree" value="1"/>
    </group>
</launch>
